### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Nikesh2712**

### CODE REVIEW

**Comment by rpanchal1996** [comment]
The resource links are very generic. Please add more specific resources.

**Comment by AvaisP** [comment]
@Nikesh2712 
Add the frontend subsection within the Web section and not after the Git section. Use typora to edit README. Also please give more specific links for codecademy and FCC

File: README.md [hunk 1]
```diff
@@ -119,3 +119,19 @@ Note: This list has been ~~shamelessly plagiarized~~ sourced from University of
 ### CheatSheet
 - [Git Ready Reckoner](https://drive.google.com/open?id=0B0TMPJ2DXuRLcWJLc3FMZDF2NU0)
 
+### Front end Development
```
Comment (AvaisP) [inline README.md]: This should be within the Web section

File: README.md [hunk 2]
```diff
@@ -119,3 +119,19 @@ Note: This list has been ~~shamelessly plagiarized~~ sourced from University of
...
+    
+    These are some very good websites to start with the basic front-end designing.
+    
+1.https://www.freecodecamp.org/
+
+2.https://www.codecademy.com/
+
+3.https://in.udacity.com/course/intro-to-html-and-css--ud304/?
+
+For bootstrap
```
Comment (AvaisP) [inline README.md]: Bootstrap should be a section within Frontend

**Comment by rpanchal1996** [comment]
duplicate

**Pull request closed without merging**